### ISSUE TITLE
Popup with html renders blank map

### ISSUE THREAD

**Issue by pjandrews** [python-visualization/folium#836]
#### Python Code

mymap = folium.Map(location=[lat_mean, lon_mean],zoom_start=14,tiles='Stamen Toner')
...
popup = folium.Popup(pout)
...
ddfg[scenario].add_child(folium.CircleMarker([lat,lon],popup=popup,radius=10,color=color,fill_color=color,fill=True))

#### Problem description

"pout" from the above code is html that has been constructed elsewhere. When pout is something like this: 

<html>           <body>             <h2>My Scenario</h2>                     <li>msg_type: 111 (0x11) (Deactivate EPS bearer context request)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- context accept Msg</li> <li>msg_type: 111 (0xc1) (Deactivate )</li> <li>Log Id: 0xb111, Name:  Incoming Message -- request Msg</li> <li>Log Id: 0x1111, Name: Incoming Message --  request Msg</li> <li>msg_type: 111 (0xc1) (Deactivate  request)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- Deactivate  Msg</li> <li>msg_type: 111 (0xcc) (Deactivate  accept)</li> <li>bearer_id: 1 (0x1) ( value 1)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- disconnect request Msg</li> <li>Log Id: 0xb111, Name: Incoming Message -- Deactivate  request Msg</li> <li>Log Id: 0xb11, Name: Outgoing Message --  accept Msg</li> <li>msg_type: 111 (0x11) ( accept)</li> <li>msg_type: 111 (0xc1) ( context request)</li>            </body>         </html>

...which folium turns into this:

"bubblingMouseEvents": true,
  "color": "#075FBE",
  "dashArray": null,
  "dashOffset": null,
  "fill": true,
  "fillColor": "#075FBE",
  "fillOpacity": 0.2,
  "fillRule": "evenodd",
  "lineCap": "round",
  "lineJoin": "round",
  "opacity": 1.0,
  "radius": 10,
  "stroke": true,
  "weight": 3
}
                ).addTo(feature_group_3481df76d3474e3898d84950ae8dc7f2);

            var popup_768ef23d8a164d6591289ca2a848b0a1 = L.popup({maxWidth: '300'});

                var html_c28edf7b450644a0b62f5c2084b9d45c = $('<div id="html_c28edf7b450644a0b62f5c2084b9d45c" style="width: 100.0%; height: 100.0%;">         <html>           <body>             <h2>My Scenario</h2>                     <li>msg_type: 111 (0x11) (Deactivate EPS bearer context request)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- context accept Msg</li> <li>msg_type: 111 (0xc1) (Deactivate )</li> <li>Log Id: 0xb111, Name:  Incoming Message -- request Msg</li> <li>Log Id: 0x1111, Name: Incoming Message --  request Msg</li> <li>msg_type: 111 (0xc1) (Deactivate  request)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- Deactivate  Msg</li> <li>msg_type: 111 (0xcc) (Deactivate  accept)</li> <li>bearer_id: 1 (0x1) ( value 1)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- disconnect request Msg</li> <li>Log Id: 0xb111, Name: Incoming Message -- Deactivate  request Msg</li> <li>Log Id: 0xb11, Name: Outgoing Message --  accept Msg</li> <li>msg_type: 111 (0x11) ( accept)</li> <li>msg_type: 111 (0xc1) ( context request)</li>            </body>         </html>         </div>')[0];
                popup_768ef23d8a164d6591289ca2a848b0a1.setContent(html_c28edf7b450644a0b62f5c2084b9d45c);

            circle_marker_7430bca8e820473bba2f33ecaec025e6.bindPopup(popup_768ef23d8a164d6591289ca2a848b0a1);

The result is that the map is blank or does not render. 

#### Expected Output
The expected output is that the map would render and that the html would appear when I click on the circle marker.

#### Output of ``folium.__version__``
'0.5.0'

**Comment by Conengmo** [comment]
I'm unable to reproduce your problem. Please provide a self-contained code snippit that reproduces it. 

I tried this and got a correct output:
```
m = folium.Map((0, 0), tiles='cartodbpositron')

html = '<html><body><h2>My Scenario</h2><li>msg_type: 111 (0x11) (Deactivate EPS bearer context request)</li><li>Log Id: 0xb111, Name: Outgoing Message -- context accept Msg</li> <li>msg_type: 111 (0xc1) (Deactivate )</li> <li>Log Id: 0xb111, Name:  Incoming Message -- request Msg</li> <li>Log Id: 0x1111, Name: Incoming Message --  request Msg</li> <li>msg_type: 111 (0xc1) (Deactivate  request)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- Deactivate  Msg</li> <li>msg_type: 111 (0xcc) (Deactivate  accept)</li> <li>bearer_id: 1 (0x1) ( value 1)</li> <li>Log Id: 0xb111, Name: Outgoing Message -- disconnect request Msg</li> <li>Log Id: 0xb111, Name: Incoming Message -- Deactivate  request Msg</li> <li>Log Id: 0xb11, Name: Outgoing Message --  accept Msg</li> <li>msg_type: 111 (0x11) ( accept)</li> <li>msg_type: 111 (0xc1) ( context request)</li></body></html></div>'

popup = folium.Popup(html)
folium.CircleMarker([0, 0], popup=popup, radius=10, color='green',
                    fill_color='red', fill=True).add_to(m)
m.save('map.html')
```

**Comment by pjandrews** [comment]
Thanks Conengmo, I figured out that some of the strings from my data are in in python list format which cause the silent error.

If I have a string that looks like this for example: "['1','2']" then I need to pre-process it before adding it to a popup.

So I'm just removing some of those characters and it works:
mystring = re.sub("[|]|'","",mystring)

**Comment by Conengmo** [comment]
Good to hear that you found the error. Since this problem was not with folium, could you then close this issue?

**Comment by pjandrews** [comment]
Sure thing. Thanks Conengmo.... another question.. I would like to use the search plugin that will be released with 0.6.0. Do you know if the release will happen soon, or if there is a way to access this plugin beforehand?

**Comment by jtbaker** [comment]
@pjandrews I'm super interested in the 0.6 release as well. Loving this library as I've been integrating it into some Dash products. Wish I wrote more JS so that I could help to contribute.
